taskcluster/libctc-package.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -xe
+
+source $(dirname "$0")/../tc-tests-utils.sh
+
+mkdir -p ${TASKCLUSTER_ARTIFACTS} || true
+
+tar -cf - \
+    -C ${DS_TFDIR}/bazel-bin/native_client/ libctc_decoder_with_kenlm.so | pixz -9 > "${TASKCLUSTER_ARTIFACTS}/libctc.tar.xz"
```

Comment:
nit: libctc is a bad name for this package, I'd go with decoder.tar.xz (and similarly, decoder-package.sh, etc).

Reply:
Done